docs/src/config.md

```diff
@@ -21,7 +21,7 @@ debugging_tc: true
 surface_setup: Bomex
 turbconv_case: Bomex
 t_end: 6hours
-turbconv: "edmf"
+turbconv: edmf
```

Comment:
This was still working, correct?

Reply:
yes, both `"edmf"` and `edmf` get parsed to the String `edmf`, so the quotes aren't needed.